recipes-config/systemd-resource-control/systemd-resource-control.bb

```diff
@@ -29,6 +30,7 @@ do_install_append() {
     # Install overrides for services
     install -d ${D}${systemd_unitdir}/system/neptune.service.d/
     install -m 0644 ${WORKDIR}/service.bluetooth.conf ${D}${systemd_unitdir}/system/connman.service.d/
+    install -m 0644 ${WORKDIR}/service.brcm43438.conf ${D}${systemd_unitdir}/system/connman.service.d/
```

Comment:
brcm43438 is only relevant for RPI?

Reply:
Yes, let's wait with BSP specific services for now.
Furthermore, bluetooth.conf shall be installed into bluetooth.service.d/ not connman.service.d.